src/react-components/room-info-dialog.js

```diff
@@ -1,118 +1,95 @@
-            : "";
-
-        return (
-          <div className={styles.attribution} key={`${a.url} ${i}`}>
-            <div className={styles.attributionName}>
-              <a href={a.url} target="_blank" rel="noopener noreferrer">
-                {a.name}
-              </a>
-            </div>
-            <div className={styles.attributionAuthor}>
-              by {a.author} {source}
-            </div>
-          </div>
-        );
-      } else {
-        return (
-          <div className={styles.attribution} key={`${a.name} ${a.author} {i}`}>
-            <div className={styles.attributionName}>{a.name}</div>
-            <div className={styles.attributionAuthor}>by {a.author}</div>
-          </div>
-        );
-      }
-    };
+function SceneAttribution({ attribution }) {
+  if (attribution.url) {
+    const source = attribution.url.includes("sketchfab.com")
+      ? "on Sketchfab"
+      : attribution.url.includes("poly.google.com")
+        ? "on Google Poly"
+        : "";
 
-    let attributions = null;
-    let creator = null;
-
-    if (hasScene && this.props.scene.attributions) {
-      creator = this.props.scene.attributions.creator;
-      attributions = (
-        <div>
-          {this.props.scene.attributions.content && this.props.scene.attributions.content.map(toAttributionDiv)}
+    return (
+      <li className={styles.attribution}>
+        <div className={styles.attributionName}>
+          <a href={attribution.url} target="_blank" rel="noopener noreferrer">
+            {attribution.name}
+          </a>
         </div>
-      );
-    }
-
-    const title = (
-      <div className={styles.title}>
-        {hasDescription && hasScene ? <FormattedMessage id="room-info.title" /> : this.props.hubName}
-      </div>
+        <div className={styles.attributionAuthor}>
+          by {attribution.author} {source}
+        </div>
+      </li>
     );
-
+  } else {
     return (
-      <DialogContainer title={title} wide={true} {...this.props}>
-        {hasDescription && (
-          <>
-            {hasScene && <div className={styles.hubTitle}>{this.props.hubName}</div>}
-            <div className={styles.description}>{this.props.hubDescription}</div>
-          </>
-        )}
+      <li className={styles.attribution}>
+        <div className={styles.attributionName}>{attribution.name}</div>
+        <div className={styles.attributionAuthor}>by {attribution.author}</div>
+      </li>
+    );
+  }
+}
+
+// TODO: Move to /room folder
+export function RoomInfoDialog({ scene, store, hubName, hubDescription, onClose }) {
+  const hasScene = !!scene;
+  const showSceneLink = hasScene && allowDisplayOfSceneLink(scene, store);
+  const attributions = (scene && scene.attributions && scene.attributions.content) || [];
+  const creator = scene && scene.attributions && scene.attributions.creator;
+
+  return (
+    <Sidebar title={<FormattedMessage id="room-info.title" />} beforeTitle={<CloseButton onClick={onClose} />}>
```

Comment:
Here I think you want to use the `intl.formatMessage` function to get a string instead of the object created by putting `FormatMessage` in jsx. https://formatjs.io/docs/react-intl/api/#usage

Reply:
You can actually pass react elements in as props. I changed the `title` propType to `PropTypes.node` so that it stops warning on this. I think we should probably use the `<FormattedMessage />` unless you explicitly require a string or if you need to use `defineMessages`. Perhaps you ware confused by the `title` prop though? It's not setting the browser's `title` prop. It's actually the title of the sidebar.